docs/reference/ml/df-analytics/apis/evaluateresources.asciidoc

```diff
@@ -13,6 +13,14 @@ Evaluation configuration objects relate to the <<evaluate-dfanalytics>>.
   (object) Defines the type of evaluation you want to perform. The value of this 
   object can be different depending on the type of evaluation you want to 
   perform. For example, it can contain <<binary-sc-resources>>.
+  
+`query`::
+  (object) The {es} query domain-specific language (<<query-dsl,DSL>>). This 
```

Comment:
It might be worth explaining here that the evaluation will only apply to documents of the `index` that match the `query`.